chef/cookbooks/neutron/recipes/common_agent.rb

```diff
@@ -206,14 +214,27 @@
   ml2_mech_drivers = neutron[:neutron][:ml2_mechanism_drivers]
   ml2_type_drivers = neutron[:neutron][:ml2_type_drivers]
 
+
   case
   when ml2_mech_drivers.include?("openvswitch")
-    # package is already installed
 
+    # package is already installed
     neutron_agent = node[:neutron][:platform][:ovs_agent_name]
     agent_config_path = "/etc/neutron/plugins/openvswitch/ovs_neutron_plugin.ini"
     interface_driver = "neutron.agent.linux.interface.OVSInterfaceDriver"
-    external_network_bridge = "br-public"
+    if !multiple_external_networks
```

Comment:
I can refactor a bit, you are right. I still need and if/else block because external_network_bridge should not be set if multiple external networks are used